src/components/Page.js

```diff
@@ -2,12 +2,24 @@
 import React, { useEffect } from 'react';
 import { Helmet } from 'react-helmet';
 import PropTypes from 'prop-types';
+import { makeStyles } from '@material-ui/styles';
 
 import useRouter from '../utils/useRouter';
 
+const useStyles = makeStyles((theme) => ({
+  pageInner: {
+    display: 'flex',
+    flexFlow: 'column nowrap',
+    alignItems: 'flex-start',
```

Comment:
On the Network not supported page, the flex-start is not centering it. If this property is removed it appears correctly there but I don't know the side effects on the other places. 

![image](https://user-images.githubusercontent.com/5901730/79106829-8556f100-7d73-11ea-86ba-9560a957142d.png)

Reply:
Also good spot, know the fix shot